static/start

```diff
@@ -1,281 +1,29 @@
-}
-
-create_and_apply_template() {
-    local route=${1:-}
-    local tag=${2:-}
-
-    if [ -z "$route" ]; then
-        echo "No route given"
-        exit 1
-    fi
-
-    echo "---------------------------------------------------------------------"
-    echo "Creating Tempate ${SYNDESIS_TEMPLATE}"
-    create_openshift_resource \
-        "install/${SYNDESIS_TEMPLATE}.yml" \
-        "$tag"
-    echo "---------------------------------------------------------------------"
-    echo
-    echo "---------------------------------------------------------------------"
-    echo "Creating new-app from ${SYNDESIS_TEMPLATE}"
-    oc new-app "${SYNDESIS_TEMPLATE}" \
-      -p ROUTE_HOSTNAME="${route}" \
-      -p OPENSHIFT_MASTER="$(oc whoami --show-server)" \
-      -p OPENSHIFT_PROJECT="$(oc project -q)" \
-      -p SAR_PROJECT="$(oc project -q)" \
-      -p OPENSHIFT_OAUTH_CLIENT_SECRET="$(oc sa get-token syndesis-oauth-client)"
-    echo "---------------------------------------------------------------------"
-}
-
-create_openshift_resource() {
-    local resource=${1:-}
-    local tag=${2:-}
-
-    oc create -f https://raw.githubusercontent.com/syndesisio/syndesis/${tag:-master}/${resource}
-}
-
-wait_for_syndesis_to_be_ready() {
-    echo "---------------------------------------------------------------------"
-    echo "Waiting for Syndesis to become ready"
-
-    # Wait a bit to start image fetching
-    # Patch imagestreams from "DockerImage" to "ImageStreamTag"
-    oc get pods -w &
-    watch_pid=$!
-    for dc in "syndesis-server" "syndesis-ui" "syndesis-meta"; do
-        echo "Waiting for $dc to be started"
-        local replicas="$(oc get dc $dc -o jsonpath='{.status.availableReplicas}')"
-        while [ "$replicas" -lt 1 ]; do
-            echo "Sleeping 10s ..."
-            sleep 10
-            replicas=$(oc get dc $dc -o jsonpath="{.status.availableReplicas}")
-        done
-    done
-    kill $watch_pid
-    echo "---------------------------------------------------------------------"
-}
-
-open_url() {
-    local url=$1
-    echo "---------------------------------------------------------------------"
-    echo "Opening $url"
-    local cmd="$(probe_commands open xdg-open chrome firefox)"
-    if [ -z "$cmd" ]; then
-        echo "Cannot find command for opening URL:"
-        echo $url
-        exit 1
-    fi
-    exec $cmd $url
-    echo "---------------------------------------------------------------------"
-}
-
-probe_commands() {
-    for cmd in $@; do
-      local ret=$(which $cmd 2>/dev/null)
-      if [ $? -eq 0 ]; then
-          echo $ret
-          return
-      fi
-    done
-}
+read -p "This install will reset your current MiniShift. OK to continue? " -n 1 -r
+echo    # (optional) move to a new line
+if [[ $REPLY =~ ^[Yy]$ ]]
+then
 
+  wget https://github.com/syndesisio/syndesis/archive/master.zip
+  unzip ./master.zip 'syndesis-master/tools/bin/*'
+  if [ -d ".syndesis" ]; then
+    rm -fr .syndesis
```

Comment:
Please  note that ~/.syndesis is also used when installing the camel-k operator. So I dont think it should be unconditionally removed. Just remove the part which is affected by the scripts.